### PR TITLE
[branch-0.7] missed try catch.

### PR DESCRIPTION
**Pull request by cloverhearts**
### What is this PR for?
Currently we can not build on branch-0.7
The cause is a missed try catch.
So, I fixed this part.

### What type of PR is it?
Hot Fix

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java [hunk 1]
```diff
@@ -338,7 +338,11 @@ public void removeNote(String id, AuthenticationInfo subject) {
       note = notes.remove(id);
       folders.removeNote(note);
     }
-    interpreterSettingManager.removeNoteInterpreterSettingBinding(subject.getUser(), id);
+    try {
+      interpreterSettingManager.removeNoteInterpreterSettingBinding(subject.getUser(), id);
+    } catch (IOException e) {
+      logger.error(e.toString(), e);
```
Comment (1ambda) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java]: Is it ok not to return here? If we get errors while removing interpreter setting binding. (LINE 342)

Comment (cloverhearts) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java]: Your opinion is worth thinking about.
However, the intent of this fix is to make the build successful,
It is to unify the code of the master branch and branch-0.7.
Therefore, you can not modify it in this PR.
Sorry

Comment (cloverhearts) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java]: ref on master branch - https://github.com/apache/zeppelin/blob/master/zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java#L342

Comment (Leemoonsoo) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java]: Let's merge this PR as a hotfix and then address improvement in another PR.

**Comment by cloverhearts** [comment]
merge

**Pull request closed without merging**